### ISSUE TITLE
Reminder: Coordinate with Frontend and Curation Team - SV Release

### ISSUE THREAD

**Issue by sheridancbio** [knowledgesystems/pipelines-scrum#793]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
Once our backend PR is merged into cbioportal codebase, will need to propagate this into production
1) No more fusion events in db (deleting studies/reimporting studies with fusions/sv data) - Curation task
- at this point, studies might show in an "in-between" state in the portal
2) Full release (no more fusion table/records inside portal, all SV) - Devops/task

These queries should both return 0 before rollout:
select count(*) from mutation_event where MUTATION_TYPE = "Fusion";
select count(*) from genetic_profile where GENETIC_ALTERATION_TYPE="FUSION";

3) Seems like the SV table is not properly linking samples to genes when the gene is not on a gene panel.  For example, the following patient has an EML4-ALK fusion.  When the "primary" gene is ALK, a sample is linked properly, but when the "primary" gene is EML4, the sample is not linked.

Example: [https://cbioportal.mskcc.org/patient?studyId=mskimpact&caseId=P-0027119](https://cbioportal.mskcc.org/patient?studyId=mskimpact&caseId=P-0027119)

<img width="599" alt="PastedGraphic-1" src="https://user-images.githubusercontent.com/366003/150200290-288f2f66-1117-43cf-ba41-c52b6e2cb960.png">

You will see the event properly displayed on the sample page:
https://cbioportal.mskcc.org/patient?studyId=mskimpact&sampleId=P-0027119-T03-IM6

![image](https://user-images.githubusercontent.com/366003/150200580-cfc12134-c636-46de-8128-bf35677af52e.png)
